cellcounter/cc_kapi/static/js/counter.js

```diff
@@ -635,19 +637,23 @@ function load_keyboard(keyboard_id) {
     }
 }
 
-function set_keyboard_primary(keyboard_id) {
+function set_keyboard_primary(keyboard_href) {
```

Comment:
I think there is an argument to refactor the naming of `set_keyboard_primary` for consistency